README.md

```diff
@@ -44,9 +58,20 @@ This template is the result of learnings from many previous projects and should
 Eventually, you can remove any unused files, such as the standalone directory or irrelevant github workflows for your project.
 Feel free to replace the License with one suited for your project.
 
-To cleanly separate the library and subproject code, the outer `CMakeList.txt` only defines the library itself while the tests and other subprojects are self-contained in their own directories. 
+To cleanly separate the library and subproject code, the outer `CMakeList.txt` only defines the library itself while the tests and other subprojects are self-contained in their own directories.
 During development it is usually convenient to [build all subprojects at once](#build-everything-at-once).
 
+### Build and test with cmake preset
+
+```bash
+cd test
+cmake --preset=test
```

Comment:
@jenisys now we have a preset which works!

Reply:
CMAKE_EXPORT_COMPILE_COMMANDS is not needed here, removed.

The other are set to have a defined Standard set for all directories. At least for